storage/src/nodestore/alloc.rs

```diff
@@ -145,10 +146,58 @@ pub fn area_size_to_index(n: u64) -> Result<AreaIndex, Error> {
         })
 }
 
-/// Objects cannot be stored at the zero address, so a [`LinearAddress`] is guaranteed not
-/// to be zero. This reserved zero can be used as a [None] value for some use cases. In particular,
-/// branches can use `Option<LinearAddress>` which is the same size as a [`LinearAddress`]
-pub type LinearAddress = NonZeroU64;
+// pub type LinearAddress = NonZeroU64;
```

Comment:
This type needs some documentation:
```suggestion
/// A newtype wrapper around an offset into the database file for a node or free list item.
/// Only the header is stored at location 0, so we use None as Option<LInearAddress> to
/// represent that a child is not present in a branch.
```